Formula/links.rb

```diff
@@ -20,22 +21,16 @@ class Links < Formula
   end
 
   depends_on "pkg-config" => :build
-  depends_on "jpeg"
-  depends_on "librsvg"
-  depends_on "libtiff"
   depends_on "openssl@1.1"
 
-  def install
-    args = %W[
-      --disable-debug
-      --disable-dependency-tracking
-      --prefix=#{prefix}
-      --mandir=#{man}
-      --with-ssl=#{Formula["openssl@1.1"].opt_prefix}
-      --without-lzma
-    ]
+  uses_from_macos "bzip2"
+  uses_from_macos "zlib"
```

Comment:
Bottles so far have been built with SSL, BZ2, and ZLIB support. Did add revision bump in case I missed something with dependencies.

e.g. Linux
```console
$ objdump -p /home/linuxbrew/.linuxbrew/Cellar/links/2.27/bin/links | grep NEEDED
  NEEDED               libbz2.so.1.0
  NEEDED               libz.so.1
  NEEDED               libssl.so.1.1
  NEEDED               libcrypto.so.1.1
  NEEDED               libc.so.6
```

macOS Monterey:
```console
% otool -L /usr/local/Cellar/links/2.27/bin/links
/usr/local/Cellar/links/2.27/bin/links:
	/usr/lib/libSystem.B.dylib (compatibility version 1.0.0, current version 1311.100.3)
	/usr/lib/libbz2.1.0.dylib (compatibility version 1.0.0, current version 1.0.8)
	/usr/lib/libz.1.dylib (compatibility version 1.0.0, current version 1.2.11)
	/usr/local/opt/openssl@1.1/lib/libssl.1.1.dylib (compatibility version 1.1.0, current version 1.1.0)
	/usr/local/opt/openssl@1.1/lib/libcrypto.1.1.dylib (compatibility version 1.1.0, current version 1.1.0)
```